Addresses question on public cert use case

stefsch opened the pull request:
Added text and reference to doc article about uploading and using a public cert in a web app.

azurecla commented:
Hi __@johnwildes__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@johnwildes : Thanks for your contribution to the Azure documentation! The author, @rayne-wiselman, has been notified to review your proposed change.

rayne-wiselman commented:
Link update is ok

rayne-wiselman commented:
@PRmerger

rayne-wiselman commented:
Hi John, please close this PR and I'll make the change in a new one. Since the PR is old trying to resolve the merge conflict will involve too many files that have changed in the interim

rayne-wiselman commented:
Here's the PR that's been opened instead - https://github.com/Azure/azure-content-pr/pull/32919. Please close.

stefsch committed:
Addresses question on public cert use case

Added text and reference to doc article about uploading and using a public cert in a web app.
